SPECS/dhcp/dhcp.spec

```diff
@@ -38,8 +39,14 @@ The ISC DHCP Client, dhclient, provides a means for configuring one or more netw
 
 
 %prep
-%autosetup -p1
-
+%setup -q
+%patch1 -p1
+tar xf bind/bind.tar.gz
+pushd bind-9.11.14
+%patch2 -p1
+popd
+tar zcf bind/bind.tar.gz bind-9.11.14
+rm -rf bind-9.11.14
 %setup -qn %{name}-%{version}
```

Comment:
Same request as in the sister PR #5892: let's remove the `%setup -qn %{name}-%{version}` line.

Reply:
Thanks @PawelWMS for catching this bug which is existing since long time. I have removed the additional %setup that is been used and used %autosetup instead.
Also changing the scope of this PR to just fix this bug and hence modified the title accordingly.